agent/app/agent_capability.go

```diff
@@ -458,7 +458,11 @@ func (agent *ecsAgent) appendServiceConnectCapabilities(capabilities []*ecs.Attr
 		}
 	}
 
```

Comment:
More details in the comment for `getSupportedAppnetInterfaceVersion` - I think we should let loader go through the list of supported appnet versions and keep trying until it successfully loads a tar, and it returns that version.